SConstruct

```diff
@@ -8,6 +10,23 @@ env = Environment(ENV=os.environ)
 env["ENV"]["OMP_NUM_THREADS"] = "1"  # Disable threading; we're parallelising at a higher level
 
 
+TESTDATA_ROOT = env.ProductDir("testdata_latiss_cpp")
+PKG_ROOT = env.ProductDir("ci_cpp_gen3")
+REPO_ROOT = os.path.join(PKG_ROOT, "DATA")
+num_process = GetOption('num_jobs')
+
+# Load exposure lists from testdata repo, to ensure consistency.
+with open(os.path.join(TESTDATA_ROOT, "raw", "manifest.yaml")) as f:
+    exposureDict = yaml.safe_load(f)
+
+# Define the collections here, so they can be used by name below.
+RAW_COLLECTION = 'raws/latiss'
```

Comment:
Is there a good reason for overriding the default here and in CALIB_COLLECTION? It might be a little confusing since everywhere else it's going to be `LATISS/calib` and  `LATISS/raw/all`  (which are the default for ingest-raws and write-curated-calibrations and there's even a class method on `lsst.obs.lsst.Latiss` for calculating them.

Reply:
I think this comes from some of my precursor tests.  Switching to the defaults.